### DISCUSSION TITLE
Why don't build the Web version?

### DISCUSSION THREAD

**Discussion by jarvislin94** [jthrilly/netCanvas#1192] [General]
Dear team,

I have a question for you: why don't build the web version of your apps? For example, the interviewer app. I think the web version is easier to be accessed by interviewers. Then they don't need to install the app on different platforms. 

Are there any concerns?

Bests
Jackson

**Comment by jthrilly** [comment]
Hi @JackZong 

Sorry for taking so long to get back to you - I've been away on holiday for the past couple of weeks. Hope its fine that I moved this to a discussion.

To answer your question: we are definitely interested in building a web based version of these apps! We have been actively applying for funding to do so for quite a while at this point. There are preliminary efforts underway internally to gauge feasibility and map out the work required. We hope to be able to share some positive news in the near term, so please look out for an announcement on our newsletter.

In the meantime, since all the software is fully open source, we welcome any effort you or others would like to give towards making it possible. PRs are always welcome! It was always the plan to make transitioning to the browser as simple as possible. Our core technology stack is 100% browser compatible.

As for why we didn't do this to begin with, the answer is that we needed functionality that simply was not available in a web browser when this project was begun in 2016. Some of it still isn't available now (PWA support on iOS, and file system APIs for browsers are still not ready). The software is designed to be run by Interviewers on their personal or organisational devices, and so the choice of which platform to install on is not really a big deal.

Hope this helps.
